auto_cpufreq/core.py

```diff
@@ -78,7 +78,28 @@ def get_config(config_file=''):
     return get_config.dict
 
 # get distro name
-dist_name = distro.id()
+try:
+    dist_name = distro.id()
+except PermissionError:
+    # Current work-around for distros like Pop!_OS where symlink causes permission issues
+    print("Warning: Cannot get distro name")
+    if os.path.exists("/etc/pop-os/os-release"):
+        print("Pop!_OS detected")
+        print("Pop!_OS uses a symbolic link for the os-release file, this causes issues and can be fixed by converting to a hard link")
+        print("Attempting to change symlink to hard link for /etc/os-release -> /etc/pop-os/os-release")
+
+        yN = input("Continue? [y/N] ")
+        if yN.lower() == "y":
```

Comment:
```suggestion
        if yN.lower() == "y":
            print("Creating hard link for /etc/os-release")
```

Reply:
After pressing "commit changes" it does nothing 🤦‍♂️